src/VisualStudio/IntegrationTest/TestUtilities/OutOfProcess/TextViewWindow_OutOfProc.cs

```diff
@@ -97,8 +98,22 @@ public void InvokeCodeActionList()
             _instance.Workspace.WaitForAsyncOperations(Helper.HangMitigatingTimeout, FeatureAttribute.SolutionCrawler);
             _instance.Workspace.WaitForAsyncOperations(Helper.HangMitigatingTimeout, FeatureAttribute.DiagnosticService);
 
-            ShowLightBulb();
-            WaitForLightBulbSession();
+            try
+            {
+                ShowLightBulb();
+                WaitForLightBulbSession();
+            }
+            catch (InvalidOperationException)
```

Comment:
> When the session is canceled instead of completed we throw an InvalidOperationException

This makes me sad :( It should be a cancellation exception right?

Reply:
who is 'we' here?

Reply:
hrmm.  i see:

```
                if (DateTimeOffset.Now > startTime + Helper.HangMitigatingTimeout)
                {
                    throw new InvalidOperationException("Expected a light bulb session to appear.");
                }
```

But this seems appropriate.  if we invoked the lightbulb... and it didn't appear. then we have a product bug.  retrying it legit seems to be maskign things.